### PR TITLE
P6 radio

### PR DESCRIPTION
**Pull request by vuongc**

### CODE REVIEW

File: src/components/atoms/p6-radio/p6-radio.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    &::before {
+      content: "";
+      display: inline-block;
+      vertical-align: top;
+      width: 10px;
+      height: 10px;
+      border: 1px solid $input-border-color;
+      background-color: $input-background-color;
+      border-radius: 50%;
+      margin-right: 5px;
```
Comment (vincentv) [inline src/components/atoms/p6-radio/p6-radio.scss:52]: like the checkbox, we could use an utf-8 character like ●

File: src/components/atoms/p6-radio/p6-radio.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  componentWillLoad() {
+    this.isChecked = this.checked;
+  }
+
+  componentDidLoad() {
+    this._input.disabled = this.disabled;
+    this._input.readOnly = this.readonly;
+  }
```
Comment (vincentv) [inline src/components/atoms/p6-radio/p6-radio.tsx:55]: I prefer the solution with the attribute configured in the render. This prevents you from searching where and when to specify the value. It also avoids modifying the input after the rendering phase.

Comment (vuongc) [reply src/components/atoms/p6-radio/p6-radio.tsx]: done

File: src/components/atoms/p6-radio/p6-radio.scss (hunk 1 above)
Comment (vuongc) [reply src/components/atoms/p6-radio/p6-radio.scss]: Using ut-8 characters for the border and the checked status

File: src/components/atoms/p6-input/p6-input.tsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  @Listen("focusout")
+  lostFocusHandler() {
+    this.checkValidity();
+    this.hasError = !!this.validationMessage;
+  }
+
+  componentDidLoad() {
+    this.checkValidity();
+    this.hasError = !!this.validationMessage;
+  }
```
[the remark this answers is not in the document]
Comment (vuongc) [reply src/components/atoms/p6-input/p6-input.tsx]: Resulting of a bad rebase

File: src/components/atoms/p6-radio/p6-radio.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import { Component, Element, Host, h, Prop, State } from '@stencil/core';
+
+@Component({
+  tag: 'p6-radio',
+  styleUrl: './p6-radio.scss',
+  scoped: true,
+})
+export class P6Radio {
+
+  @Element() host: HTMLDivElement;
```
[the remark this answers is not in the document]
Comment (vuongc) [reply src/components/atoms/p6-radio/p6-radio.tsx]: done

**Pull request merged**